### ISSUE TITLE
Dummy options in the "Study field" user field

### ISSUE THREAD

**Issue by balkian** [eestec/eestec.portal#37]
We should have more fields here. Probably related to data migration from the old Plone.

**Comment by zupo** [comment]
Nope, they are hard-coded and should be copied from the old codebase: http://svn.plone.org/svn/collective/eestec.base/trunk/eestec/base/config.py
